src/main/java/runge/wordledictionary/WordleDictionary.java

```diff
@@ -0,0 +1,59 @@
+package runge.wordledictionary;
+
+import java.io.*;
+import java.util.*;
+
+public class WordleDictionary
+{
+    private ArrayList<String> definitionSplit;
+    private ArrayList<String> wordSplit;
+
+    private final BufferedReader bufferedReader;
+
+
+    public WordleDictionary() throws IOException
+    {
+        wordSplit = new ArrayList<>();
+        definitionSplit = new ArrayList<>();
+
+        File dictionary = new File("src/main/java/runge/wordledictionary/dictionary.txt");
+        bufferedReader = new BufferedReader(new FileReader(dictionary));
+
+        if (dictionary.exists())
```

Comment:
I'm pretty sure if the file doesn't exists it will already throw an error on line 20